PyDSS/valiate_settings.py

```diff
@@ -0,0 +1,136 @@
+        },
+        "Helics": {
+            'Co-simulation Mode': {'type': bool, 'Options': [True, False]},
+            'Iterative Mode': {'type': bool, 'Options': [True, False]},
+            'Max co-iterations': {'type': int, 'Options': range(1, 1000)},
+            'Error tolerance': {'type': float},
+            'Federate name': {'type': str},
+            'Broker': {'type': str},
+            'Broker port': {'type': int},
+            'Time delta': {'type': float},
+            'Core type': {'type': str},
+            'Uninterruptible': {'type': bool, 'Options': [True, False]},
+            'Helics logging level': {'type': int, 'Options': range(0, 10)},
+        },
+        "Logging": {
+            'Logging Level': {'type': str, 'Options': ["DEBUG", "INFO", "WARNING" , "ERROR"]},
+            'Log to external file': {'type': bool, 'Options': [True, False]},
+            'Display on screen': {'type': bool, 'Options': [True, False]},
+            'Clear old log file': {'type': bool, 'Options': [True, False]},
+            'Pre-configured logging': {'type': bool, 'Options': [True, False]},
+        },
+        "MonteCarlo": {
+            'Number of Monte Carlo scenarios': {'type': int},
+        },
+        "Plots": {
+            'Create dynamic plots': {'type': bool, 'Options': [True, False]},
+            'Open plots in browser': {'type': bool, 'Options': [True, False]},
+        },
+        "Project": {
+            'Project Path': {'type': str},
+            'Start Year': {'type': int, 'Options': range(1970, 2099)},
+            'Start Day': {'type': int, 'Options': range(0, 365)},
+            'Start Time (min)': {'type': float, 'Options': range(0, 1440)},
+            'End Day': {'type': int, 'Options': range(0, 365)},
+            'End Time (min)': {'type': float, 'Options': range(0, 1440)},
+            'Date offset': {'type': int, 'Options': range(0, 365)},
+            'Step resolution (sec)' : {'type': float},
+            'Max Control Iterations' : {'type': int},
+            'Error tolerance': {'type': float},
+            'Simulation Type': {'type': str, 'Options': ["QSTS", "Dynamic", "Snapshot", "Monte Carlo"]},
+            'Active Project': {'type': str},
+            'Scenarios': {'type': list},
+            'Active Scenario': {'type': str},
+            'DSS File': {'type': str},
+            'DSS File Absolute Path': {'type': bool, 'Options': [True, False]},
+            'Return Results': {'type': bool, 'Options': [True, False]},
+            'Control mode': {'type': str, 'Options': ["Static", "Time"]},
+            'Disable PyDSS controllers': {'type': bool, 'Options': [True, False]},
+        },
+        "Profiles": {
+            "Use profile manager":  {'type': bool, 'Options': [True, False]},
+            "Profile store path": {'type': str},
+            "Profile mapping": {'type': str},
+        }
+    }
+
+
+def validate_settings(dss_args):
+    global settings_dict
```

Comment:
You can delete this line.  It would only be needed if you performed `settings_dict = X` in this function.

Reply:
fixed